apps/palette-generator/index.html

```diff
@@ -121,19 +123,24 @@ <h1>Palette Generator</h1>
             <ul>
               <li>
                 If the <code>LABEL:verb</code> is not appended, the label
-                is the given underscore separated words, i.e. "indicator_(verb)".
+                is the given underscore separated words, i.e.
+                "indicator_(action)".
               </li>
             </ul>
       </li>
       <li><code>SVG:13166,";",9011:SVGLABEL:children</code> will generate the
-          an SVG image that combines the symbols that mean "child" and "plural".
-          The cell's label is "children".
-            <ul>
-              <li>
-                  If no <code>LABEL:</code>is provided, the cell will have no
-                  label.
-              </li>
-            </ul>
+          SVG image that combines the symbols that mean "child" and "plural".
```

Comment:
This explanation and the initial examples in the text area are very useful. Do you think if the use of ";" and "/" should also be explained? Such as, ";" is to superimpose the second symbol on the first one, while "/" is to have them displayed side by side.

Moreover, it would be nice to have one or more examples in the text area to show the usage of both.

Reply:
Thanks for that suggestion.  I've added more examples and found a way, although awkward, of placing Bliss symbols into the HTML.